### PR TITLE
Distil* configs

### PR DESCRIPTION
**Pull request by ayeffkay**
## Features
This PR deals with distilled Conversational RuBERT (pytorch) models which were added in the library. 

- Configuration files for Paraphraser, NER, RuSentiment and SQuAD added
- `overview.rst` was supplemented with `Distil*` models scores
- download links and models description added in `bert.rst`

### CODE REVIEW

File: deeppavlov/models/torch_bert/torch_transformers_sequence_tagger.py [hunk 1]
```diff
@@ -333,10 +333,12 @@ def load(self, fname=None):
 
         if self.pretrained_bert:
             config = AutoConfig.from_pretrained(self.pretrained_bert, num_labels=self.n_classes,
-                                                output_attentions=False, output_hidden_states=False)
+                                                output_attentions=False, output_hidden_states=False, 
+                                                return_dict=False)
             self.model = AutoModelForTokenClassification.from_pretrained(self.pretrained_bert, config=config)
         elif self.bert_config_file and Path(self.bert_config_file).is_file():
             self.bert_config = AutoConfig.from_json_file(str(expand_path(self.bert_config_file)))
+            self.bert_config.return_dict = False
```
Comment (slowwavesleep) [inline deeppavlov/models/torch_bert/torch_transformers_sequence_tagger.py:341]: I suggest not hard-coding this sort of thing. Moreover, using the newer data structure that is returned with return_dict = True doesn't pose much of a problem. You can access relevant elements with dot notation (like `output.loss` etc). Additionally, this appears to be a more robust approach, as opposed to indexing outputs with some magic numbers (like output[0]).

File: deeppavlov/models/torch_bert/torch_bert_squad.py [hunk 2]
```diff
@@ -150,24 +149,26 @@ def __call__(self, features: List[InputFeatures]) -> Tuple[List[int], List[int],
...
-        input_masks = [f.attention_mask for f in features]
-        input_type_ids = [f.token_type_ids for f in features]
+        _input = {}
+        for elem in ['input_ids', 'attention_mask', 'token_type_ids']:
+            _input[elem] = [getattr(f, elem) for f in features]
 
-        b_input_ids = torch.cat(input_ids, dim=0).to(self.device)
-        b_input_masks = torch.cat(input_masks, dim=0).to(self.device)
-        b_input_type_ids = torch.cat(input_type_ids, dim=0).to(self.device)
+        for elem in ['input_ids', 'attention_mask', 'token_type_ids']:
```
Comment (slowwavesleep) [inline deeppavlov/models/torch_bert/torch_bert_squad.py:156]: Hard-coding this is not a good idea since some models (roberta, for example) don't use token_type_ids.

File: deeppavlov/models/torch_bert/torch_bert_squad.py [hunk 3]
```diff
@@ -150,24 +149,26 @@ def __call__(self, features: List[InputFeatures]) -> Tuple[List[int], List[int],
...
             # Forward pass, calculate logit predictions
-            outputs = self.model(input_ids=b_input_ids, attention_mask=b_input_masks, token_type_ids=b_input_type_ids)
+            _model_input = {key:value for (key,value) in _input.items() if key in self.model.forward.__code__.co_varnames}
+            outputs = self.model(**_model_input)
             logits_st, logits_end = outputs[:2]
 
-            bs = b_input_ids.size()[0]
-            seq_len = b_input_ids.size()[-1]
+            bs = _input['input_ids'].size()[0]
+            seq_len = _input['input_ids'].size()[-1]
```
Comment (slowwavesleep) [inline deeppavlov/models/torch_bert/torch_bert_squad.py:166]: You don't have to slice it. The relevant dimension can be accessed by simply passing it's index to the size method. For example, `_input['input_ids'].size(0)` would work the same as `_input['input_ids'].size()[0]`.

File: deeppavlov/models/preprocessors/torch_transformers_preprocessor.py [hunk 4]
```diff
@@ -54,10 +54,13 @@ def __init__(self,
                  vocab_file: str,
                  do_lower_case: bool = True,
                  max_seq_length: int = 512,
-                 return_tokens: bool = False,
+                 return_tokens: bool = False, 
+                 add_token_type_ids: bool = False,
```
Comment (slowwavesleep) [inline deeppavlov/models/preprocessors/torch_transformers_preprocessor.py:58]: The newer tokenizer provides relevant features (such as token_type_ids) if it's required by the defined pretrained model.

File: deeppavlov/models/preprocessors/torch_transformers_preprocessor.py [hunk 5]
```diff
@@ -54,10 +54,13 @@ def __init__(self,
...
                  **kwargs) -> None:
         self.max_seq_length = max_seq_length
         self.return_tokens = return_tokens
+        self.add_token_type_ids = add_token_type_ids
```
Comment (slowwavesleep) [inline deeppavlov/models/preprocessors/torch_transformers_preprocessor.py:62]: See the previous comment.

**Review by slowwavesleep — APPROVED** [review]
